### DISCUSSION TITLE
Why are these invalid DXF files created?

### DISCUSSION THREAD

**Discussion by cbvgn** [LibreDWG/libredwg#515] [Q&A] [answered]
File 222.dxf has a root dictionary, but not as the first entity in the OBJECTS section as required
File A2.dxf has an invalid empty line at line number 18395 and 18498, after fixing that same issue as in file 222.dxf
[invalid files.zip](https://github.com/LibreDWG/libredwg/files/9256490/invalid.files.zip)

**Comment by rurban** [comment, marked as answer]
Thanks, nice catches. But this needs to be filed as issue, not discussion.

**Reply by cbvgn** [reply]
Yes, I will
